# goal 
uint8 EndEffectorGoal = 0
uint8 JointGoal = 1
uint8 CartesianGoal = 2

uint8 type
geometry_msgs/Pose goal_pose
sensor_msgs/JointState goal_joint_state

moveit_msgs/RobotState start_state

octomap_msgs/Octomap octomap

bool execute_path

moveit_msgs/PlanningOptions planning_options
---
# result
bool success
trajectory_msgs/JointTrajectory trajectory
---
# feedback
float64 elapsed_time
